### PR TITLE
Dont force redraw on hard reset for status line

### PR DESCRIPTION
**Pull request by Yaraslaut**
Closes https://github.com/contour-terminal/contour/issues/1303

### CODE REVIEW

File: src/vtbackend/Terminal.cpp [hunk 1]
```diff
@@ -1823,17 +1823,6 @@ void Terminal::hardReset()
     _state.inputGenerator.reset();
 }
 
-void Terminal::forceRedraw(std::function<void()> const& artificialSleep)
```
Comment (christianparpart) [inline src/vtbackend/Terminal.cpp:1826]: I would probably keep this function. It may serve good also as its own action :)

**Review by christianparpart — COMMENTED** [review]
> Dont force redraw on hard reset for status line

The PR title doesn't match the actual PR change. Why? 🤔

**Comment by christianparpart** [comment]
> Build / Ubuntu Linux 22.04

This one will keep failing, we have to ignore this check (for now), and merge despite this. But the rest should get green. Weird, that it's not. (can also have a look later, if needed :) )

**Comment by Yaraslaut** [comment]
Alrigh, will save it and change how it works then

**Comment by Yaraslaut** [comment]
> > Build / Ubuntu Linux 22.04
> 
> This one will keep failing, we have to ignore this check (for now), and merge despite this. But the rest should get green. Weird, that it's not. (can also have a look later, if needed :) )
> 
> EDIT:
> 
> ```
> /Users/runner/work/contour/contour/src/vtbackend/Terminal_test.cpp:197:17: error: no member named 'forceRedraw' in 'vtbackend::Terminal'
>     mc.terminal.forceRedraw({});
>     ~~~~~~~~~~~ ^
> ```
> 
> That's why. But yeah, I'd suggest we keep this function (and make sure it's working). We can attach that functionality to its own action (`ForceRedraw`)

So i just deleted call to force redraw

**Review by christianparpart — APPROVED** [review]

**Comment by christianparpart** [comment]
> Check Changelog / Check Changelog Action (pull_request)

FYI: This is failing (and a new check :D) because you did not modify the metainfo.xml. this is a pretty dump check, it simply tests if it was subject to the PR diff. 

If no changelog entry is needed, simply add `no changelog` label to the PR to get that check ignored (go green).

**Pull request merged**